packages/core/src/tracing/esmSupportedVersion.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const semver = require('semver');
+
+/** @type {(version: string) => boolean} */
+module.exports = exports = function esmSupportedVersion(version) {
+  return semver.satisfies(version, '>=14.0.0 <=18.17.1');
```

Comment:
Question: Should we ask for <20? If there is a new Node v18 release it no longer works or?

Reply:
Updated the code to check <20